components/src/directives/dropdown-direction/index.ts

```diff
@@ -0,0 +1,34 @@
+import {Directive, DirectiveBinding} from 'vue';
+
+function fixPosition(el: HTMLElement) {
+  el.classList.remove('--positon-top');
+  el.classList.remove('--positon-bottom');
```

Comment:
can be refactored to `el.classList.remove('--positon-top', '--positon-bottom');`

Reply:
done

Reply:
Please mark resolved